packages/shared/src/components/cards/MarketingCTA/index.tsx

```diff
@@ -0,0 +1,3 @@
+export * from './MarketingCTACard';
+export * from './MarketingCTAList';
+export * from './common';
```

Comment:
This should probably not export everything, as it'll be picked up in the cards/index export too.
So this import (from the modal PR) looks like you're importing a generic card header or description
```
import { CTAButton, Description, Header, Title } from '../cards';
```

Reply:
I'll remove the common export for now, would that work?